src/Components/NFT/Detail/index.js

```diff
@@ -79,8 +79,8 @@ const Details = () => {
           <div className={styles.details__profile}>
             <div className={styles.details__profile__picture}></div>
             <div className={styles.details__user__info}>
-              <p>{nftData.owner}</p>
-              <h6>{user?.account_id}</h6>
+              <p>Creator</p>
+              <h6>{user?.full_name}</h6>
```

Comment:
Use the full name from the nftData not from the user object

Reply:
nft data's owner object is undefined and any other field has no name field 
![image](https://user-images.githubusercontent.com/43183288/149613671-b95b1466-979b-4459-aef6-ce03625dfdaa.png)

Reply:
@mrsamia Please check the API response. It's there 
![NFT-details-API-response](https://user-images.githubusercontent.com/18471335/149613820-59c88ae4-fe1a-41a9-bb0f-f7528ac36f76.PNG)

Reply:
Please check the mapNftDetails method in utils.js file, it's not mapping properly

Reply:
Thanks. I checked it. But the problem id when we get all the nft data, the api gives only the owner id. Not the whole owner object. And when we go for the details from the homepage it uses the reducer data. So what I am trying to say is If we use the reducer data we have to update the all nft fetching api with owner object. And if we always fetch data from api on details page then it will take time to load. Which one should we do. I think the all nft holding the owner profile object is a good way.

Reply:
This NFT details page can be access without any authentication so anyone can directly access this page. So we need to get the updated data from the API. 
And if you're doing any changes to mapNftDetails method, please check it's usage, I guess we're using this method in other places.